internal/api/service.go

```diff
@@ -1,30 +1,29 @@
-package service
+package api
 
 import (
+	v12 "github.com/bancodobrasil/stop-analyzing-api/internal/api/v1"
```

Comment:
Is the number "2" a typo? Maybe it was supposed to be "v1"?

Reply:
fixed in https://github.com/bancodobrasil/stop-analyzing-api/pull/21/commits/544ac955d77f285785fadbd0b5985d31c76583d5